trace_proto.go

```diff
@@ -161,6 +166,24 @@ func timestampProto(t time.Time) *timestamppb.Timestamp {
 	}
 }
 
+// copyMonitoredResourceAttributes copies proto monitoredResource to proto map field (Span_Attributes)
+// it creates the map if it is nil.
+func copyMonitoredResourceAttributes(out **tracepb.Span_Attributes, mr *monitoredrespb.MonitoredResource) {
```

Comment:
It is very unusual in Go to use pointers to pointers like this. Can you convert to a return value instead?

Reply:
So you would use it like:

```go
sp.Attributes = copyMonitoredResourceAttributes(sp.Attributes, mr)
```

Reply:
I followed existing example of copying attributes in the same file. 
Anyway, I have changed it as per your suggestion.